CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+# 92.4.0
+
+This minor release addresses issues with Dialogs in Dark Mode, adds delegate methods for Navigation Drawer, and makes Outlined Text Areas more configurable.
+
+## New deprecations
+
+This minor release deprecates Chips+ChipThemer and MDCChipViewFontThemer.
+
+## New features
+
+MDCTextInputControllerOutlinedTextArea now allows users to specify `minimumLines` and `expandsOnOverflow`.
+
+```objc
+self.textFieldController =
+      [[MDCTextInputControllerOutlinedTextArea alloc] initWithTextInput:self.textField];
+MDCTextInputControllerOutlinedTextArea *textInputControllerOutlinedTextArea =
+    (MDCTextInputControllerOutlinedTextArea *)self.textFieldController;
+textInputControllerOutlinedTextArea.minimumLines = 1;
+textInputControllerOutlinedTextArea.expandsOnOverflow = YES;
+```
+
+MDCBottomDrawerPresentationControllerDelegate has new methods that allow clients to respond to animation lifecycle events and touches to the scrim.
+
+#### Touch events set to delegate
+
+Touch events are propagated to delegate to allow clients to interpret touches in the scrim area.
+
+```
```

Comment:
Please document the language for this snippet, here and throughout.

Reply:
Done.